schema.sql

```diff
@@ -0,0 +1,287 @@
+CREATE TABLE
+  account (
+    id VARCHAR PRIMARY KEY,
+    total_donations_usd DECIMAL(10, 2),
+    total_matching_pool_allocations_usd DECIMAL(10, 2),
+    donors_count INT
+  );
+
+CREATE TABLE
+  list (
+    id VARCHAR PRIMARY KEY,
+    owner_id VARCHAR NOT NULL,
+    name VARCHAR NOT NULL,
+    description TEXT,
+    default_registration_status ENUM(
+      'Pending',
+      'Approved',
+      'Rejected',
+      'Graylisted',
+      'Blacklisted'
+    ) NOT NULL,
+    FOREIGN KEY (owner_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  list_admin (
+    list_id INT NOT NULL,
+    admin_id VARCHAR NOT NULL,
+    PRIMARY KEY (list_id, admin_id),
+    FOREIGN KEY (list_id) REFERENCES list (id),
+    FOREIGN KEY (admin_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  list_registration (
+    id SERIAL PRIMARY KEY,
+    registrant_id VARCHAR NOT NULL,
+    status ENUM(
+      'Pending',
+      'Approved',
+      'Rejected',
+      'Graylisted',
+      'Blacklisted'
+    ) NOT NULL,
+    submitted_at TIMESTAMP NOT NULL,
+    updated_at TIMESTAMP,
+    registrant_notes TEXT,
+    admin_notes TEXT,
+    tx_hash VARCHAR,
+    FOREIGN KEY (registrant_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  pot_factory (
+    id VARCHAR PRIMARY KEY,
```

Comment:
change to `SERIAL`

Reply:
Why use serial, when we can id them by the account id deployed to, since it’s unique, and it’ll  also be easy to reference,

Reply:
sure we can do that, especially since there won't be many rows in this table. i was suggesting to follow the pattern we have established elsewhere of using an int for the primary key rather than the account ID since strings (particularly ones that can be 64 chars long, like account IDs) are not very efficient as a PK for various db operations

Reply:
> id will basically be useless, can't fetch with it, as we won't know the id, for example, in donations where we need to fetch the pot donated to, we'll use the account id.

i was going to say that, but i think my sql knowledge is begging to be improved, will add the id as serial and also add the account_id, make it unique and we can index it for efficient querying.